### PR TITLE
Add Example

### PR DESCRIPTION
**Pull request by jrichardlai**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "react-native-tooltip",
   "version": "3.1.0",
   "description": "A react-native wrapper for showing tooltips",
-  "main": "ToolTipText.ios.js",
```
Comment (chirag04) [inline package.json]: I think this change was part of your prev PR.

File: ToolTipText.ios.js [hunk 2]
```diff
@@ -1,28 +0,0 @@
-'use strict';
-
-var React = require('react-native');
-var {
-  Text,
-  requireNativeComponent,
-} = React;
-
-var ViewClass = React.createClass({
```
Comment (chirag04) [inline ToolTipText.ios.js]: This was part of prev PR

**Comment by chirag04** [comment]
i think your prev PR changes got added into this.

**Comment by jrichardlai** [comment]
Yes, wanted to get the other branch merged first so the diff would not include the other commits

**Comment by jrichardlai** [comment]
@chirag04 this one should be good to go if you want to test it out.

**Pull request merged**